configs/2020.10.15.ini

```diff
@@ -0,0 +1,22 @@
+[DEFAULT]
+version=2020.10.14
```

Comment:
this should be 2020.10.15

Reply:
thanks for the thorough review, exactly what i needed

Reply:
78df98e